src/main/java/com/coalesce/ttb/gui/MaterialGui.java

```diff
@@ -0,0 +1,286 @@
+package com.coalesce.ttb.gui;
+
+import com.coalesce.gui.ItemBuilder;
+import com.coalesce.gui.PlayerGui;
+import com.coalesce.plugin.CoPlugin;
+import org.bukkit.Bukkit;
+import org.bukkit.DyeColor;
+import org.bukkit.Material;
+import org.bukkit.Sound;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.ItemStack;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.bukkit.ChatColor.*;
+
+public final class MaterialGui extends PlayerGui {
+	
+    private int currentPage;
+    private short durability;
+	private Material selection;
+	private final TextGui previousMenu;
+
+    private static final int GUI_ROWS = 6;
+    private static final int MATERIAL_ROWS = 5;
+	private static final float SOUND_VOLUME = 3;
+	private static final Sound ACTION_SOUND = Sound.BLOCK_WOOD_BUTTON_CLICK_ON;
+	private static final Sound INVALID_ACTION_SOUND = Sound.BLOCK_ANVIL_PLACE;
+    private static ArrayList<Material> validBuildMaterials;
+	
+	/**
+	 * Creates a MaterialGui for a player.
+	 * @param plugin The providing plugin.
+	 * @param selection The currently selected material.
+	 * @param previousMenu The previous menu.
+	 */
+	public MaterialGui(CoPlugin plugin, Material selection, TextGui previousMenu) {
+        super(plugin, GUI_ROWS * 9, DARK_GRAY + "Material Menu");
+
+        this.selection = selection;
+        this.currentPage = 0; //Page numbers start at 0
+		this.durability = 0;
+		this.previousMenu = previousMenu;
+
+		setupControlBar();
+		setMaterials();
+
+    }
+
+	private void setupControlBar(){
+
+		final int totalPages = (getValidBuildMaterials().size() / (MATERIAL_ROWS * 9)) + 1;
+
+		//Previous Page
+		setItem(45,
+				user -> {
```

Comment:
It's a GitHub thing. There isn't really a need to edit them... They look completely normal in the editor.